packages/@aws-cdk/aws-apigatewayv2/README.md

```diff
@@ -15,9 +15,132 @@
 ---
 <!--END STABILITY BANNER-->
 
-
 This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
 
+`aws-apigatewayv2` supports the `HTTP` API for Amazon API Gateway.
```

Comment:
This file needs a proper revamp here. Start with a nice introduction, add a section for 'HTTP APIs', break it up into further sections with code snippets. 

See how it's done in [cognito](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-cognito/README.md) or [apigateway](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-apigateway/README.md) and follow a similar pattern.

Reply:
I have updated the `README` and explains different types of APIs it support. Let me know if it works.